RecoMuon/L3MuonProducer/python/L3Muons_cfi.py

```diff
@@ -1,128 +1,62 @@
 #this is not the actual configuration of HLT
 #changing this file will not change the behavior of HLT
 #see the actual configuration in confDB
 
-L3Muons = cms.EDProducer("L3MuonProducer",
-    ServiceParameters = cms.PSet(
-        Propagators = cms.untracked.vstring('SmartPropagatorAny',
+L3Muons = _mod.L3MuonProducer.clone(
+    ServiceParameters = dict(
+        Propagators = ['SmartPropagatorAny',
             'SteppingHelixPropagatorAny',
             'SmartPropagator',
-            'SteppingHelixPropagatorOpposite'),
-        RPCLayers = cms.bool(True),
-        UseMuonNavigation = cms.untracked.bool(True)
+            'SteppingHelixPropagatorOpposite'],
     ),
-    MuonCollectionLabel = cms.InputTag("hltL2Muons","UpdatedAtVtx"),
-    TrackLoaderParameters = cms.PSet(
-        PutTkTrackIntoEvent = cms.untracked.bool(True),
-        VertexConstraint = cms.bool(False),
-        MuonSeededTracksInstance = cms.untracked.string('L2Seeded'),
-        Smoother = cms.string('KFSmootherForMuonTrackLoader'),
-        MuonUpdatorAtVertexParameters = cms.PSet(
-            MaxChi2 = cms.double(1000000.0),
-            Propagator = cms.string('SteppingHelixPropagatorOpposite'),
-            BeamSpotPositionErrors = cms.vdouble(0.1, 0.1, 5.3)
+    TrackLoaderParameters = dict(
+        PutTkTrackIntoEvent = True,
+        Smoother = 'KFSmootherForMuonTrackLoader',
+        MuonUpdatorAtVertexParameters = dict(
+            Propagator = 'SteppingHelixPropagatorOpposite',
         ),
-        SmoothTkTrack = cms.untracked.bool(False),
-        DoSmoothing = cms.bool(True),
-        beamSpot = cms.InputTag("hltOfflineBeamSpot")
+        DoSmoothing = True,
+        beamSpot = "hltOfflineBeamSpot"
     ),
-    L3TrajBuilderParameters = cms.PSet(
-        ScaleTECxFactor = cms.double(-1.0),
-        TrackerRecHitBuilder = cms.string('WithTrackAngle'),
-        MuonTrackingRegionBuilder = cms.PSet(
-            EtaR_UpperLimit_Par1 = cms.double(0.25),
-            Eta_fixed = cms.double(0.2),
-            OnDemand = cms.double(-1.0),
-            Rescale_Dz = cms.double(3.0),
-            Eta_min = cms.double(0.05),
-            Rescale_phi = cms.double(3.0),
-            EtaR_UpperLimit_Par2 = cms.double(0.15),
+    L3TrajBuilderParameters = dict(
+        TrackerRecHitBuilder = 'WithTrackAngle',
+        MuonTrackingRegionBuilder = dict(
+            #Eta_fixed = cms.double(0.2), # please check, default type is cms.bool(True)
+            Rescale_Dz = 3.0,
+            Eta_min = 0.05,
             DeltaZ_Region = cms.double(15.9),
-            Rescale_eta = cms.double(3.0),
-            PhiR_UpperLimit_Par2 = cms.double(0.2),
-            vertexCollection = cms.InputTag("pixelVertices"),
-            Phi_fixed = cms.double(0.2),
-            DeltaR = cms.double(0.2),
-            EscapePt = cms.double(1.5),
+            #Phi_fixed = cms.double(0.2), # please check, default type is cms.bool(False)
```

Comment:
This was a bug in the original config: as pointed out by @jeongeun this parameter was supposed to be a `bool`, while it was inserted as a `double`: as such the defalt `Eta_fixed=cms.bool(True)` is used instead.
Please check, and suggest a fix, if needed: otherwise the [default](https://github.com/cms-sw/cmssw/blob/bbb17baf002ae74665d12738285a64f27ba84a66/RecoMuon/L3MuonProducer/src/L3MuonProducer.cc#L254) `true` value will be retained

Reply:
`Phi_fixed=cms.bool(True)` is indeed what is used in the configuration of this producer in the HLT and it should be fixed here to reflect hat.